Formula/p/pixi.rb

```diff
@@ -34,6 +35,7 @@ class Pixi < Formula
   end
 
   def install
+    ENV["PIXI_VERSION"] = version.to_s
```

Comment:
If the motivation is nicer versions when doing `brew install --HEAD` (based on https://github.com/prefix-dev/pixi/issues/2321):
```suggestion
    ENV["PIXI_VERSION"] = Utils.safe_popen_read("git", "describe", "--tags").chomp if build.head?
```

Reply:
this results in something like `v0.34.0-4-g09f68343`. i think it would be nice to remove the `v` to make it the same as pixi's default `pixi --version` (`pixi 0.34.0`) (this will lead to nicer integration with things like https://starship.rs)

Also, for some reason it results in `0.0.6-1161-gfb85c0b` on my system when not running `git fetch --tags` first so i added that as well...